### PR TITLE
feat(docs): FRMW-158 - Migrate rxActiveUrl docs in README to ngdocs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-158

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxActiveUrl/rxActiveUrl.js [hunk 1]
```diff
@@ -4,7 +4,10 @@
  * @description
```
Comment (CITguy) [inline src/rxActiveUrl/rxActiveUrl.js]: please add `@deprecated` tag to ngdoc comment

Comment (bry4n) [inline src/rxActiveUrl/rxActiveUrl.js]: added `@deprecated`

**Comment by CITguy** [comment]
LGTM

File: src/rxActiveUrl/rxActiveUrl.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 /**
  * @ngdoc overview
  * @name rxActiveUrl
+ * @deprecated THIS COMPONENT IS MARKED TO BE REMOVED IN A FUTURE RELEASE.
  * @description
  * # rxActiveUrl Component
  *
- * [TBD]
+ * A Component adds a class name of 'selected' to an LI if the current url matches
```
Comment (100stacks) [inline src/rxActiveUrl/rxActiveUrl.js]: Still learning the Framework...what does `LI` mean?  If it means an HTML `<li>` element, it does not appear that the demo/markup example is implementing `rxActiveUrl` properly.  An example might help newbies.

Comment (CITguy) [inline src/rxActiveUrl/rxActiveUrl.js]: It does mean an HTML `<li>` element, and this component was originally built for `rxAppNav` functionality that no longer makes sense to expose. Considering that this component is deprecated (and slated for removal), I don't see much value modifying the documentation or the demo.

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**